### ISSUE TITLE
Not working at all

### ISSUE THREAD

**Issue by ghost** [inspectdev/inspect-issues#70]
- Inspect version: 0.6.8
- Electron: 12.0.7
- OS version and architecture: win32 x64 10.0.19043
- Language: undefined

Steps to Reproduce:

1.  This was working months ago during the trial, but I payed $10.00 today so that i could use it and it is just not working.  I've tried everything, including restarting both devices (surface book 3/ipad pro 2018). It worked exactly twice today, which, luckily, was enough to get a few console messages back, but I need this to work properly.  On your developer tools, the console error messages give me the following:

Main._lateInitialization: 0.197998046875 ms
​ Request Network.setAttachDebugStack failed. {"code":-32601,"message":"'Network.setAttachDebugStack' was not found","data":[{"code":-32601,"message":"'Network.setAttachDebugStack' was not found"}]}
callback @ InspectorBackend.js:1126
​ Protocol Warning: Attempted to dispatch an unspecified method 'Page.defaultAppearanceDidChange': {"method":"Page.defaultAppearanceDidChange","params":{"appearance":"Dark"}}
reportProtocolWarning @ InspectorBackend.js:60
​ Request Debugger.setAsyncCallStackDepth failed. {"code":-32602,"message":"Some arguments of method 'Debugger.setAsyncStackTraceDepth' can't be processed","data":[{"code":-32602,"message":"Parameter 'depth' with type 'Integer' was not found."},{"code":-32602,"message":"Some arguments of method 'Debugger.setAsyncStackTraceDepth' can't be processed"}]}
callback @ InspectorBackend.js:1126
​ Request Overlay.enable failed. {"code":-32601,"message":"'Overlay' domain was not found","data":[{"code":-32601,"message":"'Overlay' domain was not found"}]}
callback @ InspectorBackend.js:1126
​ Request Overlay.setShowViewportSizeOnResize failed. {"code":-32601,"message":"'Overlay' domain was not found","data":[{"code":-32601,"message":"'Overlay' domain was not found"}]}
callback @ InspectorBackend.js:1126
​ Request Profiler.enable failed. {"code":-32601,"message":"'Profiler' domain was not found","data":[{"code":-32601,"message":"'Profiler' domain was not found"}]}
2.

and so forth.  I need to debug my app, not yours so I'll leave you with this info and hopefully you can get it fixed.  I should also take a moment to bitch about how much I HATE apple and the fact that they make life as a developer suck extreme amounts of ass.  Ok I'll stop complaining here, and hope that you have a fix for this soon or can help otherwise.

Thanks,

Joe

**Comment by auchenberg** [comment]
@AdmiralKirk1701 Thanks for the report Joe. The console log messages are expected. 

What version of iOS are you running on your iPad?

**Comment by ghost** [comment]
HI!

 

I’ve just upgraded to iPadOS 14.6 in hopes that Apple would have activated WebGL 2.0 as default, but no such luck.  Still, the audioworklet fixes to Safari are critical for me and I need to be using version of ipados 14.5 or later (not that they will let me downgrade, but still… LOL)

 

Thanks so much for looking into this!

 

Joe

**Comment by ghost** [comment]
Hi Kenneth,

 

FYI, viewing the console errors on the iPad has started to work again, but getting results is not consistent.  The following errors still persist in the Developer tools console:

 

InspectorBackend.js:1301 Request Debugger.setSkipAllPauses failed. {"code":-32601,"message":"'Debugger.setSkipAllPauses' was not found","data":[{"code":-32601,"message":"'Debugger.setSkipAllPauses' was not found"}]}

 

InspectorBackend.js:1301 Request Overlay.setPausedInDebuggerMessage failed. {"code":-32601,"message":"'Overlay' domain was not found","data":[{"code":-32601,"message":"'Overlay' domain was not found"}]}

callback @ InspectorBackend.js:1301

InspectorBackend.js:65 Protocol Warning: Attempted to dispatch an unspecified method 'Debugger.globalObjectCleared': {"method":"Debugger.globalObjectCleared"}

 

 

In any case, with enough reloading/refreshing, I am eventually able to see the results on the console despite the errors.  Thanks for looking at my problem, and I hope that more improvements are made on Inspector.

 

Regards,

 

Joe

**Comment by VadimGorbenko** [comment]
same for me, so my trial period will be wasted
- Inspect version: 0.6.9
- Electron: 13.1.1
- OS version and architecture: win32 x64 10.0.19042
- Language: undefined

Steps to Reproduce:

1. Start local server
2. Connect iPad gen6 ios 14.4.2;
3. Open Inspect app;
4. Open new tab in safari with local web app (SPA on Vue.js);
5. Click in Inspect UI on tab, that was opened on prev step;
5. get nothing with error in Inspect devtools:
```
InspectorBackend.js:1301 
Request Network.setAttachDebugStack failed. {"code":-32601,"message":"'Network.setAttachDebugStack' was not found","data":[{"code":-32601,"message":"'Network.setAttachDebugStack' was not found"}]}
  callback	@	InspectorBackend.js:1301
  _onMessage	@	InspectorBackend.js:518
  _dispatchMessage	@	Connections.js:60
  dispatchEventToListeners	@	Object.js:131
  (anonymous)	@	InspectorFrontendHost.js:68
  postMessage (async)		
  s	@	devtools.tsx:62
  i	@	devtools.tsx:39
  emit	@	events.js:315
  (anonymous)	@	AppProvider.tsx:79
  emit	@	events.js:315
  onMessage	@	electron/js2c/renderer_init.js:93
```

**Comment by yannbf** [comment]
Same here. The screencast feature might be nice, but unfortunately doesn't work:

- Inspect version: 0.6.9
- Electron: 13.1.1
- OS version and architecture: darwin x64 20.5.0

Trying out on an ios simulator (iPhone 11 + ios 14.5) in XCode 12.5

![image](https://user-images.githubusercontent.com/1671563/122180507-c7ecef80-ce88-11eb-9fa2-65af08547d96.png)

The elements panel is also empty, and xcode is telling me this:
![image](https://user-images.githubusercontent.com/1671563/122180632-e357fa80-ce88-11eb-8d88-39706649e9a2.png)

**Comment by auchenberg** [comment]
The connection timing issue should have been resolved in `v0.7.0` which is rolling out as we speak.

@yannbf The issue isn't connected to screencasting, and if you look at your screenshot the Elements tab is empty which implies lack of data and the connection issue

@AdmiralKirk1701 @VadimGorbenko This error message is expected, and you should see range of them as not all DevTools APIs are implemented in Inspect. That said, I believe you were affected by this connection issue

Please give the new release a spin, and let me know how things work for you. Sorry about this.
